GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -134,6 +137,22 @@ public GVRMesh createMesh(GVRContext ctx, AiMesh aiMesh) {
             }
         }
 
+        // Vertex Colors
+        for(int c = 0; c < MAX_VERTEX_COLORS; c++) {
+            FloatBuffer fbuf = aiMesh.getColorBuffer(c);
+            if (fbuf != null) {
+                FloatBuffer coords = FloatBuffer.allocate(aiMesh.getNumVertices() * 4);
+                FloatBuffer source = aiMesh.getColorBuffer(c);
+                String name = "a_color";
```

Comment:
quick question on this.  if the model does have multiple 'vertex color' arrays (for whatever reason), it looks like they'll all get the same name.  that would be a problem, wouldn't it?